### ISSUE TITLE
[BUG] Furnitures(entity: item_frame) can be duplicated with item_frame

### ISSUE THREAD

**Issue by Xiao-MoMi** [PluginBugs/Issues-ItemsAdder#1004]
Describe the bug
Furnitures(entity: item_frame) can be duplicated with item_frame

My Config here:
huawei_mate_40_pro:
display_name: "华为Mate40Pro"
resource:
model_id: 5210
material: PAPER
generate: false
model_path: item/furniture/phone/huawei_mate_40_pro
behaviours:
furniture:
entity: item_frame
solid: false
hitbox:
length: 1
width: 1
height: 1
placeable_on:
walls: false
ceiling: false
floor: true

Expected behavior
furnitures can be duplicated with an item_frame and an item_frame will be consumed

Info (please complete the following information):
Server version: Purpur1.16.5 - 1171
Client version: Minecraft 1.16.5 with optifine

Special Info
ItemsAdder version:2.4.2
LightAPI Fork version: 3.5.1
LoneLibs version: 1.0.13
ProtocolLib version: 4.6.1

**Comment by HyperionL** [comment]
Same here, when we get the furniture item of an item frame in survival game mode it also breaks the item frame and duplicates the item inside, like a merge. Also, if you put some chair at item frame and you do right-click, you can sit on.
![2021-07-08_18 02 55](https://user-images.githubusercontent.com/64606171/124990461-6692e900-e00e-11eb-9698-2066479f9a72.png)
![2021-07-08_18 03 01](https://user-images.githubusercontent.com/64606171/124990471-6c88ca00-e00e-11eb-9678-cb0ce4f00c42.png)
![2021-07-08_18 03 40](https://user-images.githubusercontent.com/64606171/124990506-790d2280-e00e-11eb-881d-462fac5dac22.png)

**Comment by GuazP** [comment]
I guess it's not itemsAdder bug because I can't reproduce, it have to be your configuration of server to stack dropped items with same internal type. I don't recall where, but I simply removed stacking if entites aren't the same, because of keys dupe.

**Comment by HyperionL** [comment]
Tried new installation of latest paper 1.16.5 -> We can sit on chairs on item frames. Also duplicates the item with the item frame on break.
Tried new installation of latest purpur 1.16.5 -> Same.
Tried new installation of latest airplane 1.16.5 -> Same.
Tried new installation with some spigot 1.16.5 -> Same.

I've tried with the recommended version for 1.16.5 and the latest one. Same results. In creative, it is normal, you'll get the item frame and the item. The problem's with survival.

![2021-07-09_11 36 03](https://user-images.githubusercontent.com/64606171/125094906-a6ef7700-e0a1-11eb-89f2-1ed2d3bf113e.png)
![2021-07-09_11 36 06](https://user-images.githubusercontent.com/64606171/125094920-a9ea6780-e0a1-11eb-9611-a4e3feb2d4ab.png)
![2021-07-09_11 36 10](https://user-images.githubusercontent.com/64606171/125094939-ad7dee80-e0a1-11eb-9b74-3a5a5857f3ee.png)


There are only LoneLibs, ProtocolLibs, and ItemsAdder plugins at my plugin folder this time. If you use something like ClearLag with its item merging feature, all items duplicate on the drop but in this case, there are no ClearLag or similar and only happens with furniture and item frames.
Also as far as I know, in the server config files there's only possible to modify the item merge radius, and in this new case, it's the standard config of the server executable with no custom values to reduce things and help with lag.
